Vagrantfile

```diff
@@ -46,6 +46,9 @@ Vagrant.configure(VAGRANTFILE_API_VERSION) do |config|
   config.vm.box = "ubuntu/trusty64"
   config.vm.box_url = "https://vagrantcloud.com/ubuntu/trusty64/version/1/provider/virtualbox.box"
 
+  # Consistent IP for demos and tutorials (10.141.141.10 is mesosphere playa address)
+  config.vm.network :private_network, ip: "10.141.141.20"
```

Comment:
Did you mean "10.141.141.10" or "10.141.141.20" ? The comment above says "10.10.141.141.10" is mesosphere playa address.